dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1824,23 +2208,46 @@
         }
       }
     },
-    "EntityWithResolution": {
-      "type": "object",
+    "EntityWithTags": {
       "allOf": [
         {
           "$ref": "#/definitions/Entity"
         }
       ],
+      "type": "object",
+      "required": [
+        "type",
+        "tags"
+      ],
       "properties": {
-        "resolutions": {
-          "description": "The collection of entity resolution objects.",
+        "type": {
+          "type": "string",
```

Comment:
Could this be an extensible enum to improve the experience for our customers?

Reply:
We will keep for string for now. In the future we may revisit making this extensible enum

Reply:
I see. My recommendation is if you expect your customers to use this property a lot, to make it an enum sooner than later so they know what some values to expect.